core/network/nic.go

```diff
@@ -210,26 +210,18 @@ func (i *InterfaceInfo) IsVLAN() bool {
 	return i.VLANTag > 0
 }
 
-// CIDRAddress returns Address.Value combined with CIDR mask.
+// CIDRAddress returns Address.Value combined with subnet mask.
+// TODO (manadart 2020-07-02): Usage of this method should be phased out
+// in favour of calling ValueForCIDR on each member of the addresses slice.
 func (i *InterfaceInfo) CIDRAddress() (string, error) {
```

Comment:
Would it be possible to replace all calls to `CIDRAddress` with `ValueForCIDR` and remove it as part of this PR or is it too much effort? I see the deprecation comment but I guess others will not since their editor will just autocomplete the call for them.

Reply:
I don't anticipate anyone using it before it can be removed.

I would rather fix the places where it is used and remove it in one go, but that is an important functional change - this is forerunner to a patch in flight that I need to get done.

Reply:
Makes sense